app/main.py

```
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.strip().split()
+    if len(parts) != 3:
+        return
+    cmd, src, dest = parts
+    if cmd != "cp" or src == dest:
+        return
+    try:
+        with open(src, "r") as file_in, open(dest, "w") as file_out:
+            file_out.write(file_in.read())
+    except FileNotFoundError:
+        return
```

Issue: The checklist requires raising a custom `FileNotFound` exception when the source file does not exist, but here you are just returning. You should raise your custom `FileNotFound` exception instead of returning in the `except FileNotFoundError` block.

Checklist reference: 'Raise a custom FileNotFound exception (inheriting from Exception) if the source file does not exist.'